src/PIL/TiffImagePlugin.py

```diff
@@ -949,28 +949,34 @@ def load(self, fp: IO[bytes]) -> None:
             warnings.warn(str(msg))
             return
 
+    def _get_ifh(self):
+        ifh = self._prefix + self._pack("H", 43 if self._bigtiff else 42)
+        if self._bigtiff:
+            ifh += self._pack("HH", 8, 0)
+        ifh += self._pack("Q", 16) if self._bigtiff else self._pack("L", 8)
```

Comment:
See https://web.archive.org/web/20240706160214/https://www.awaresystems.be/imaging/tiff/bigtiff.html for a description of the header values.